### PR TITLE
Fix database table errors on build

### LINKED ISSUE #926: Type "privacy" does not exist
On first attempt at `docker-compose up` I received the following error:

```ERROR:  type "privacy" does not exist```

Here is the docker output and surrounding context:

```postgres_1   | ### importing table schemas
postgres_1   | /var/run/postgresql:5432 - accepting connections
postgres_1   | CREATE ROLE
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | CREATE INDEX
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | CREATE INDEX
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | CREATE INDEX
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | CREATE INDEX
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | CREATE INDEX
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | CREATE INDEX
postgres_1   | CREATE TABLE
postgres_1   | GRANT
postgres_1   | 2019-11-21 21:42:39.903 UTC [172] ERROR:  type "privacy" does not exist at character 200
postgres_1   | 2019-11-21 21:42:39.903 UTC [172] STATEMENT:  CREATE TABLE public.playlists
postgres_1   | 	(
postgres_1   | 	    title text,
postgres_1   | 	    id text primary key,
postgres_1   | 	    author text,
postgres_1   | 	    description text,
postgres_1   | 	    video_count integer,
postgres_1   | 	    created timestamptz,
postgres_1   | 	    updated timestamptz,
postgres_1   | 	    privacy privacy,
postgres_1   | 	    index int8[]
postgres_1   | 	);
postgres_1   | ERROR:  type "privacy" does not exist
postgres_1   | LINE 10:     privacy privacy,
postgres_1   |                      ^
postgres_1   | 2019-11-21 21:42:39.903 UTC [172] ERROR:  relation "public.playlists" does not exist
postgres_1   | 2019-11-21 21:42:39.903 UTC [172] STATEMENT:  GRANT ALL ON public.playlists TO kemal;
postgres_1   | ERROR:  relation "public.playlists" does not exist
postgres_1   | 2019-11-21 21:42:39.960 UTC [183] ERROR:  relation "playlists" does not exist
postgres_1   | 2019-11-21 21:42:39.960 UTC [183] STATEMENT:  CREATE TABLE playlist_videos
postgres_1   | 	(
postgres_1   | 	    title text,
postgres_1   | 	    id text,
postgres_1   | 	    author text,
postgres_1   | 	    ucid text,
postgres_1   | 	    length_seconds integer,
postgres_1   | 	    published timestamptz,
postgres_1   | 	    plid text references playlists(id),
postgres_1   | 	    index int8,
postgres_1   | 	    live_now boolean,
postgres_1   | 	    PRIMARY KEY (index,plid)
postgres_1   | 	);
postgres_1   | ERROR:  relation "playlists" does not exist
postgres_1   | 2019-11-21 21:42:39.962 UTC [183] ERROR:  relation "public.playlist_videos" does not exist
postgres_1   | 2019-11-21 21:42:39.962 UTC [183] STATEMENT:  GRANT ALL ON TABLE public.playlist_videos TO kemal;
postgres_1   | ERROR:  relation "public.playlist_videos" does not exist
postgres_1   | CREATE TYPE
postgres_1   | ### invidious database setup finished
```

I suspect this may be a cascade of errors starting here (earlier messages indicating errors connecting):

```
postgres_1   | 2019-11-21 21:42:30.280 UTC [51] LOG:  shutting down
postgres_1   | 2019-11-21 21:42:30.295 UTC [49] LOG:  database system is shut down
postgres_1   |  done
postgres_1   | server stopped
postgres_1   |
postgres_1   | PostgreSQL init process complete; ready for start up.
postgres_1   |
postgres_1   | 2019-11-21 21:42:30.383 UTC [7] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres_1   | 2019-11-21 21:42:30.383 UTC [7] LOG:  listening on IPv6 address "::", port 5432
postgres_1   | 2019-11-21 21:42:30.386 UTC [7] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres_1   | 2019-11-21 21:42:30.398 UTC [58] LOG:  database system was shut down at 2019-11-21 21:42:30 UTC
postgres_1   | 2019-11-21 21:42:30.403 UTC [7] LOG:  database system is ready to accept connections
invidious_1  | Unhandled exception:  (DB::ConnectionRefused)
invidious_1  |   from lib/sqlite3/src/sqlite3/connection.cr:8:11 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
invidious_1  | Caused by: Cannot establish connection (PQ::ConnectionError)
invidious_1  |   from lib/pg/src/pq/connection.cr:34:9 in 'new'
invidious_1  |   from lib/pg/src/pg/connection.cr:13:23 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
invidious_1  | Caused by: Error connecting to 'postgres:5432': Connection refused (Errno)
invidious_1  |   from /usr/lib/crystal/core/socket.cr:70:11 in 'new'
invidious_1  |   from lib/pg/src/pg/connection.cr:13:23 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
postgres_1   | 2019-11-21 21:42:33.464 UTC [65] FATAL:  role "kemal" does not exist
invidious_1  | Unhandled exception:  (DB::ConnectionRefused)
invidious_1  |   from lib/sqlite3/src/sqlite3/connection.cr:8:11 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
invidious_1  | Caused by: role "kemal" does not exist (PQ::PQError)
invidious_1  |   from lib/pg/src/pq/connection.cr:162:7 in 'read'
invidious_1  |   from lib/pg/src/pq/connection.cr:157:7 in 'connect'
invidious_1  |   from lib/db/src/db/database.cr:57:9 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
invidious_invidious_1 exited with code 1
postgres_1   | 2019-11-21 21:42:37.409 UTC [66] FATAL:  role "kemal" does not exist
invidious_1  | Unhandled exception:  (DB::ConnectionRefused)
invidious_1  |   from lib/sqlite3/src/sqlite3/connection.cr:8:11 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
invidious_1  | Caused by: role "kemal" does not exist (PQ::PQError)
invidious_1  |   from lib/pg/src/pq/connection.cr:162:7 in 'read'
invidious_1  |   from lib/pg/src/pq/connection.cr:157:7 in 'connect'
invidious_1  |   from lib/db/src/db/database.cr:57:9 in '->'
invidious_1  |   from lib/db/src/db/pool.cr:255:3 in 'build_resource'
invidious_1  |   from lib/db/src/db/pool.cr:380:11 in 'build_database'
invidious_1  |   from /usr/lib/crystal/core/slice.cr:241:13 in '???'
invidious_1  |   from /usr/lib/crystal/core/crystal/once.cr:255:3 in '__crystal_once'
invidious_1  |   from src/invidious.cr:44:1 in '__crystal_main'
invidious_1  |   from /usr/lib/crystal/core/crystal/main.cr:47:14 in 'main'
invidious_1  |   from ???
```

### PR DESCRIPTION
**Pull request by m4teh**
Move privacy on top, preventing errors during database build. Fixes #926 #962

### CODE REVIEW

File: docker/Dockerfile.postgres [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM postgres:10
+FROM postgres:10.11-alpine
```
Comment (leonklingele) [inline docker/Dockerfile.postgres]: Why is this change required? It will break the database when upgrading from the non-alpine image. See https://github.com/omarroth/invidious/pull/687

**Comment by m4teh** [comment]
Odd. I thought I only did a pull request for the postgres.sh. Going to close for now.
